### ISSUE TITLE
Site not loading, lots of downloads

### ISSUE THREAD

**Issue by bbuske** [subhra74/xdm#577]
Hi, 

I have noticed a pretty weird issue: Whenever I am going to visit the website https://forum.thresholdx.net/ or any of its subpages, like login, XDM tries to download tons of files and scripts and the site does not completely load. It seems to be loading without any CSS and JS.  

This is the only website, where I have noticed the issue, and it renders the site completely unusable. The only way to access it, is to disable the extension from XDM.

**Comment by subhra74** [comment]
Can you remove .GZ file extension in settings and check if that helps?

**Comment by bbuske** [comment]
Yes, that works. But it can only be considered a workaround because the setting is global and as such other tar.gz files won't download anymore. I am wondering, why from that forums it wants to download the .gz files, but not from others... I am visiting several boards of thos. After all, it is a quite common forum software.

**Comment by bykidi** [comment]
Bumping this.
Some sites pack their javascript files into gz archives which leads to breaking the site and getting ton of downloads.
Please exclude .js.gz (at least) by default.
